app/scripts/lib/transaction/metrics.ts

```diff
@@ -1085,34 +1030,26 @@ async function buildEventFragmentProperties({
   if (transactionContractMethod === contractMethodNames.APPROVE) {
     properties = {
       ...properties,
-      // TODO: Fix in https://github.com/MetaMask/metamask-extension/issues/31860
-      // eslint-disable-next-line @typescript-eslint/naming-convention
+      simulation_sending_assets_total_value:
+        properties.simulation_sending_assets_total_value ??
+        // TODO: Remove when we have fiatValue in transactionMeta
+        // https://github.com/MetaMask/core/pull/6178
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (transactionMeta as any).fiatValue,
```

Comment:
Minor, this is what preview builds are perfect for, to check your client CI before merging the core PR.